components/core/JsInterop/interop.ts

```diff
@@ -43,8 +46,8 @@ export function addDomEventListener(element, eventName, invoker) {
   if (element == 'window') {
     window.addEventListener(eventName, callback);
   } else {
-    let dom = getDom(element);
-    (dom as HTMLElement).addEventListener(eventName, callback);
+      let dom = getDom(element);
```

Comment:
We need to indent only two characters at here.

Reply:
fixed